work_items/kyc-vcs/requirements-to-accept-a-kyc-vc.md

```diff
@@ -0,0 +1,590 @@
+
+
+<table>
+  <tr>
+   <td><strong>Vocabulary</strong>
+   </td>
+   <td><strong>Field Description & References</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Method:
+   </td>
+   <td>Documentary, non-documentary
+   </td>
+  </tr>
+  <tr>
+   <td>Source:
+   </td>
+   <td>Photo ID, public records, official document, other document (<em>we need to determine the full list of acceptable documents</em>), note that this field is a high level description of the source, the document type is included below
+   </td>
+  </tr>
+  <tr>
+   <td>Match Result:
+   </td>
+   <td>Full match (all data elements matched), partial match (3 out of 4 elements matched), match with issues (2 out of 4 elements matched), unmatched <em>(the issuer should not issue a VC based on an unmatched result)</em>
+   </td>
+  </tr>
+  <tr>
+   <td>Unmatched Data:
+   </td>
+   <td>Name, date of birth, address, TaxID or ID Number <em>(could be one or many)</em>. It is important to provide information on unmatched data, this does not make the VC invalid, but rather demonstrates the quality of the verification process
+   </td>
+  </tr>
+  <tr>
+   <td>Verifier Name:
+   </td>
+   <td>Free text (examples Jumio, Onfido, Prove, etc.) suggest using the name here instead of a DID so that the Verifier does not need to resolve the DID information
+   </td>
+  </tr>
+  <tr>
+   <td>Document Type:
+   </td>
+   <td>Drivers license, passport, permanent resident card, voter registration card, government ID, military ID, state ID card, n/a (for non documentary)
+   </td>
+  </tr>
+  <tr>
+   <td>Issuing State/Province:
+   </td>
+   <td>Full name of state or province or n/a if not applicable
+   </td>
+  </tr>
+  <tr>
+   <td>Issuing Country:
+   </td>
+   <td>Use the <a href="https://www.iso.org/iso-3166-country-codes.html">ISO</a> country abbreviation or full country name or n/a
+   </td>
+  </tr>
+  <tr>
+   <td>ID Number:
+   </td>
+   <td>Document ID number, format will depend on ID type and country
+   </td>
+  </tr>
+  <tr>
+   <td>Expiry Date:
+   </td>
+   <td>Universal Date Format (<a href="https://www.iso.org/iso-8601-date-and-time-format.html">ISO 8601</a>): YYYY/MM/DD (do we want to allow n/a?). Photo ID documents should usually have an expiry, but other documents such as birth certificates, utility bills, etc. will not)
+   </td>
+  </tr>
+</table>
+
+
+#### Additional Considerations
+
+4. We may want to consider all possible sources for document verification even if they are atypical such as: social security card, utility bill, or birth certificate for example.
+5. It is common for companies to compare KYC information to the information on an ID, this includes full name, date of birth and address. Therefore, we may want to consider how to replicate this process with verifiable credentials. To some extent this will be covered in the “Match Result”.
```

Comment:
```suggestion
6. It is common for companies to compare KYC information to the information on an ID, this includes full name, date of birth and address. Therefore, we may want to consider how to replicate this process with verifiable credentials. To some extent this will be covered in the “Match Result”.
```